api/wasm/cpp/sdk_container.sh

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
+set -e
+
 # basics
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
 apt-get upgrade -y
 apt-get install -y --no-install-recommends apt-utils ca-certificates
 apt-get autoremove -y
 apt-get clean
-apt-get install -y --no-install-recommends software-properties-common apt-transport-https git wget curl libglib2.0-dev autoconf autotools-dev automake libtool cmake python
+apt-get install -y --no-install-recommends software-properties-common apt-transport-https git wget curl pkg-config autoconf autotools-dev automake libtool cmake python zlib1g-dev
```

Comment:
Just curious what caused these changes?

Reply:
I removed `libglib2.0-dev` which wasn't necessary (it used to be needed by V8, but it's not anymore, and I'm not sure why it was here in the first place), so I had to add `pkg-config` and `zlib1g-dev` directly, since those are needed and were previously installed as transitive dependencies of `libglib2.0-dev`.